.circleci/common.sh

```diff
@@ -90,7 +94,7 @@ function install_deps_pytorch_xla() {
 function build_torch_xla() {
   XLA_DIR=$1
   pushd "$XLA_DIR"
-  CC=clang-9 CXX=clang++-9 python setup.py install
+  CC=gcc-7 CXX=gcc-7 python setup.py install
```

Comment:
This change actually broke upstream CI for `pytorch/pytorch`, see https://github.com/pytorch/pytorch/runs/4651120454?check_suite_focus=true

What was the reasoning behind the compiler switch?

Reply:
@wonjoolee95 Can we use clang-9 for all of the compilation task for pt/xla?